cloudwatch-alarms.tf

```diff
@@ -8,8 +8,8 @@ resource "aws_cloudwatch_metric_alarm" "low_memory" {
   statistic           = "Maximum"
   threshold           = "100"
   alarm_description   = "Database instance memory above threshold"
-  alarm_actions       = var.alarm_sns_topics
-  ok_actions          = var.alarm_sns_topics
+  alarm_actions       = [var.alarm_sns_topics]
```

Comment:
I think `alarm_sns_topics` is a list already,  pls look at the variables file 
module is expecting `[sns1,sns2]`   if you are changing it for string type you should update the variables file as well.

Reply:
done